<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Rajiv" ignored="0" name="Suite 02" tests="3" failures="1" timestamp="02 Feb 2019 05:48:37 GMT" time="129.64" errors="0">
  <testcase name="validateLoginImage" time="1.492" classname="qa.crm.testcases.LoginPageTest"/>
  <testcase name="validateLoginPageTitle" time="0.032" classname="qa.crm.testcases.LoginPageTest">
    <failure type="java.lang.AssertionError" message="expected [#1 Free CRM software in the cloud for sales and service] but found [Cogmento says…]">
      <![CDATA[java.lang.AssertionError: expected [#1 Free CRM software in the cloud for sales and service] but found [Cogmento says…]
	at qa.crm.testcases.LoginPageTest.validateLoginPageTitle(LoginPageTest.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- validateLoginPageTitle -->
  <testcase name="LoginTest" time="1.221" classname="qa.crm.testcases.LoginPageTest"/>
</testsuite> <!-- Suite 02 -->
